app/latigo/executor/__init__.py

```diff
@@ -219,18 +219,18 @@ def _fetch_sensor_data(self, task: Task) -> typing.Optional[SensorDataSet]:
                     spec, time_range
                 )
                 if not sensor_data:
-                    logger.warning(f"Error getting sensor data: {err}")
+                    logger.error(f"Error getting sensor data: {err}")
                 elif not sensor_data.ok():
-                    logger.warning(f"Sensor data '{sensor_data}' was not ok")
+                    logger.error(f"Sensor data '{sensor_data}' was not ok")
             else:
-                logger.warning(
+                logger.error(
                     f"Error getting spec for project={project_name} and model={model_name}"
                 )
         except Exception as e:
             logger.error(
-                f"Could not fetch sensor data for task '{task.project_name}.{task.model_name}': {e}"
+                f"Could not fetch sensor data for task '{task.project_name}.{task.model_name}': {e}",
```

Comment:
```suggestion
                "Could not fetch sensor data for task '%s.%s':", task.project_name, task.model_name,
```
Logging documentation suggests delayed strings substitution, and pass the values as parameters. Also there is no need to add the exception instance to the message since with `exc_info=True` it will be provided it anyway.